SPECS/lua/lua.spec

```diff
@@ -1,104 +1,146 @@
 %define LICENSE_PATH LICENSE.PTR
-%define majmin %(echo %{version} | cut -d. -f1-2)
+%global major_version 5.4
+
+# If you are incrementing major_version, enable bootstrapping and adjust accordingly.
+# Version should be the latest prior build. If you don't do this, RPM will break.
+%global bootstrap 0
+%global bootstrap_major_version 5.3
+%global bootstrap_version %{bootstrap_major_version}.6
 
 Summary:        Programming language
 Name:           lua
-Version:        5.3.5
-Release:        8%{?dist}
+Version:        5.4.3
+Release:        1%{?dist}
 License:        MIT
-URL:            https://www.lua.org
+URL:            http://www.lua.org
```

Comment:
Why revert to HTTP? HTTPS seems to be working.

Reply:
fixed